.circleci/config.yml

```diff
@@ -6,6 +6,15 @@ parameters:
     type: string
     default: go1.15.10-ec819fab057e2efdcf478fec05bb5533957721ec
 
+commands:
```

Comment:
Are there any options for downloading a pre-built `git-cliff` so we don't have to install rust here and build it locally?

Reply:
The GitHub project publishes tarballs with shasums and detached signatures, though it's a bit unclear to me at the moment where the public signing key is located.

Additionally, they public a Docker image via GitHub's container registry, but it's unclear to me at the moment how to verify the contents of this image in similar fashion to the tarball.

I agree that the rust build can be improved, but I'm not sure it's worth blocking this PR for it, since a) this logic mirrors what we have going in the other branches, aiding in maintainability, and b) the `changelog` job happens in parallel with other tests and such, and the hold-up it produces should be minimal.

My feeling would be that using some kind of pre-built package should be left for future work, since using Git is both correct and secure.

Reply:
Makes sense to me!

Reply:
I'll put up an issue for this. Thanks for the +1!